src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -110,6 +114,14 @@ private static Person createEditedPerson(ReadOnlyPerson personToEdit,
         return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
     }
 
+    /**
+     * Raise events for editing a person to update display.
+     */
+    private void raisePersonEditedEvents(ReadOnlyPerson editedPerson) {
+        EventsCenter.getInstance().post(new PersonModifiedEvent(editedPerson));
+        EventsCenter.getInstance().post(new JumpToListRequestEvent(model.getIndex(editedPerson)));
```

Comment:
Are you sure raising them together is a good idea? If you do this, why not keep them in a single event? We create separate events only when we have cases for success or failures. Raising them together is the same as raising a single event.